packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -316,7 +316,7 @@ class WebpackCLI {
             return this.builtInOptionsCache;
         }
 
-        const minimumHelpFlags = [
+        const minimumHelpFlags = new Set([
```

Comment:
Perf here would have such little impact that there's no point. If there is 10000 objects I understand, but this is Max 30, so O(30ns) vs O(30^2) wouldnt make much difference

Reply:
we did a lookup for every flag, there could be ~300 flags